### ISSUE TITLE
Feature Request: Change Preview/Live Layout and Behavior

### ISSUE THREAD

**Issue by lightning-software** [quelea-projection/Quelea#351]
The preview and live panels layout and method of advancing seems clunky to me, especially for less tech-savvy users. I would like for the process to be streamlined and more user-friendly. This is my proposal:

1. Make the preview pane act as both the preview and live controller pane.
2. Make the live pane (the far right) a display pane with a preview show and a live show (preview on top), labeled as to which is which.

As for behavior:
-- when an item (say lyrics) are loaded into the preview pane, a single click displays the item into the preview show, a right-click brings up a context menu for editing and whatever else can be done, and a double-click displays the item in the preview AND live shows.
-- In the schedule, a double-click on an item loads it into the preview pane and displays it in the preview and live shows (like a double-click on an item in the preview pane).

All other shortcuts remain in place.

I don't know the names of the pieces that display the preview and live output, so I called them "shows". The preview pane would now be more of a control panel, as one can control the preview and live shows and edit in place as needed.

Basically, streamline the process of making something live, which will be more user-friendly. Also, possibly make this layout/functionality optional in the settings.

I am trying to program this myself, and am reaching out to you per contribution instructions. However, I am not extremely familiar with Java nor github, so it's not easy for me....

**Comment by berry120** [comment]
It's not clear to me that this will necessarily make things clearer or less clunky:

 - With the current behaviour you know that anything you change on the live pane has an immediate result on the live view. With this new system you have one "control pane", which, depending on your actions, will affect either the preview or live panes (with it not being immediately clear you have to double-click something to put it live.)
 - Does this mean you need to double-click every time you want to change verses in a live setting? We definitely couldn't have that - most people I know use the arrow keys to advance slides in the live view rather than clicking, and I'm not sure this would be possible with this setup either.
 - I'm not sure how this would work at all with videos or images - they don't have text sections, so don't really work in the same way.

Bear in mind Quelea already has a good number of regular users who are used to the way it works currently, so major changes like this need a *lot* of careful consideration, and we'd only consider adding them if there were clear benefits, or clear evidence of a big problem with the way things are currently. I'm not sure this proposal fits into this category unfortunately, so I'm afraid (unless I'm completely misunderstanding) it's probably not a PR we'd accept.
